src/components/NpsSurvey.js

```diff
@@ -42,12 +42,12 @@ const NpsSurvey = () => {
   }
 
   useEffect(() => {
-    if (registrationStatus === 'registered') {
+    if (registrationStatus === 'registered' && acceptedTos) {
```

Comment:
Without this, before the ToS is accepted we get this error in the console, which cases the test to fail (but with no indication of why):

asyncToGenerator.js:6 Uncaught (in promise) 
Response {type: 'cors', url: 'https://terra-rex-dev.appspot.com/api/npsResponses/lastTimestamp', redirected: false, status: 403, ok: false, …}
body: (...)
bodyUsed: false
headers: Headers {}
ok: false
redirected: false
status: 403
statusText: ""
type: "cors"
url: "https://terra-rex-dev.appspot.com/api/npsResponses/lastTimestamp"
[[Prototype]]: Response

The fact that the error is uncaught (as opposed to caught and displayed to the user like we normally do) seems to be the issue.